### ISSUE TITLE
[PlayState] Add a setting to remember each x minutes that you have paused your current game

### ISSUE THREAD

**Issue by BanCrash** [darklinkpower/PlayniteExtensionsCollection#257]
### Extension name

PlayState

### Feature Description

Hi, I find very useful having a setting that if it's enabled when you have your current game paused for x minutes, it will send a notification to remember you that it's paused. So in case it's no longer paused (in case you are only pausing playtime) but you forgot to resume you can do it after seeing the notification.

I have this feature in my previous game launcher and it's funny how many times I resumed a game that I was already playing thanks to the notification because I forgot to do so. So for me it's very useful.

**Comment by darklinkpower** [comment]
The extension has functionality to automatically suspend and resume when the game window is not in the foreground and can be enabled in settings. Does that help?

**Comment by BanCrash** [comment]
> The extension has functionality to automatically suspend and resume when the game window is not in the foreground and can be enabled in settings. Does that help?

Yep, I know, but sometimes I just pause the game without minimizing it, because I just leave the computer so not necessary to minimize the game haha.
